app/models/payloads/github_pull_request.rb

```diff
@@ -59,7 +59,7 @@ def branch_name
     end
 
     def base_branch_master?
-      pull_request&.dig('base', 'ref') == 'master'
+      pull_request&.dig('base', 'ref') == 'master' || pull_request&.dig('base', 'ref') == 'main'
```

Comment:
```suggestion
      %w(main master).include? pull_request&.dig('base', 'ref')
```
That's probably more idiomatic Ruby?

Reply:
yep, even better, thanks 👍